runtime/worker.rs

```diff
@@ -491,6 +505,55 @@ impl MainWorker {
       validate_import_attributes_cb: Some(Box::new(
         validate_import_attributes_callback,
       )),
+      enable_code_cache: options.v8_code_cache.is_some(),
+      eval_context_code_cache_cbs: options.v8_code_cache.map(|cache| {
+        let cache_clone = cache.clone();
+        let modified_timestamp_getter =
+          options.modified_timestamp_getter.clone();
+        let modified_timestamp_getter_clone =
+          options.modified_timestamp_getter.clone();
```

Comment:
This closure is quite big - I suggest to extract it to a separate function call, not a big deal though